.cargo/config.toml

```diff
@@ -18,11 +18,18 @@ rustflags = ["-C", "target-feature=+crt-static"]
 [target.i686-pc-windows-msvc]
 rustflags = ["-C", "target-feature=+crt-static"]
 
-[target.aarch64-apple-darwin]
+[target.'cfg(target_os = "macos")']
 linker = "rust-lld"
 
-[target.aarch64-unknown-linux-gnu]
-linker = "aarch64-linux-gnu-gcc"
+[target.'cfg(all(target_os = "linux", target_env = "gnu"))']
+rustflags = [
+  "--cfg",
+  "tokio_unstable",
+  "-Zshare-generics=y",
+  "-Zthreads=8",
+  "-Csymbol-mangling-version=v0",
+  "-Clink-arg=-fuse-ld=lld",
```

Comment:
What about `rust-lld`?

Reply:
I did try using `rust-lld` (both here and with `linker = "rust-lld"`). It didn't work claiming that `rust-lld` isn't on the PATH. I'm not sure why it works on other platforms but not here.

Either way, it shouldn't matter as this is temporary until rustc makes lld default everywhere. It was just (today!) [made default for x86-64](https://blog.rust-lang.org/2024/05/17/enabling-rust-lld-on-linux.html), but we also care about build speeds on aarch64.

Reply:
Ah, might have to extend the workaround in `crates/napi/build.rs`, but it's fine for now